src/System.Data.SqlClient/src/System/Data/RecordsAffectedEventArgs.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+
+//------------------------------------------------------------------------------
+
+using System;
+
+namespace System.Data
+{
+    public sealed class StatementCompletedEventArgs : System.EventArgs
```

Comment:
The using is redundant as well :smile:, since we're in a namespace that includes "System".